File: eCompany.Models/Chart.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace eCompany.Models
{
    public class Chart
    {
        public string[] labels { get; set; }
        public List<Datasets> datasets { get; set; }
    }

    public class Datasets
    {
        public string label { get; set; }
        public int data { get; set; }
        public string backgroundColor { get; set; }
        public string borderColor { get; set; }
        public int borderWidth { get; set; }
        

    }
}

File: eCompany/wwwroot/js/chart.js
$(document).ready(function () {
    var id = document.getElementById("EmployeeId").textContent;
    var c = document.getElementById("vbar_chart");
    var ctx = c.getContext("2d");
    var tData = $.getValues(`/Admin/Task/GetTasksChart?id=${id}`);
    const allData = tData.datasets.map(getData);
    var myBarChart = new Chart(ctx, {
        type: 'bar',
        data: {
            labels: tData.labels,
            datasets: [{
                label: "Number of Tasks",
                backgroundColor: tData.datasets.map(getColors),
                data: tData.datasets.map(getData),
                borderColor: tData.datasets.map(getBorderColor),
                borderWidth: 1
            }]
        },
        options: {
            layout: {
                padding: {
                    top: 10,
                }
            },
            plugins: {
                title: {
                    display: true,
                    text: "Employee's Tasks Chart "
                }
            },
            maintainAspectRatio: false,
            scales: {
                yAxes: [{
                    ticks: {
                        beginAtZero: true
                    }
                }]
            }

        },

    });



});



function getData(item) {
    return item.data;
}

function getColors(item) {
    return item.backgroundColor;
}

function getBorderColor(item) {
    return item.borderColor;
}